Sources/MapboxMaps/Foundation/Extensions/Encodable+ToJSON.swift

```diff
@@ -21,7 +21,13 @@ internal extension Encodable {
     /// - Throws: Errors occurring during conversion.
     /// - Returns: A string with JSON representing the object.
     func toString(encoding: String.Encoding = .utf8) throws -> String {
-        let data = try JSONEncoder().encode(self)
+        let data: Data
+        if #available(iOS 13.0, *) {
+            data = try JSONEncoder().encode(self)
+        } else {
+            // JSONEncoder doesn't support fragments on older iOS versions
```

Comment:
Can you please add a link from the PR description here? It was very helpful to understand why it doesn't work.